### ISSUE TITLE
Imou looc camera - not able to get stream

### ISSUE THREAD

**Issue by arkpoah** [ccrisan/motioneye#2049]
Hello,

I have the stream url which works well in vlc, but when I add it in motioeye, I only get the no camera symbol.
My config is : 
```
netcam_url rtsp://192.168.1.76:554/cam/realmonitor?channel=1&subtype=1
netcam_userpass admin:password
```
the `frame Rate Dimmer` has a value greater than 0.
I also test to add `netcam_use_tcp` to `off`

I see nothing special in logs :/

**Comment by starbasessd** [comment]
How are you adding the camera? Using the WebGUI or directly editing the .conf files?

**Comment by arkpoah** [comment]
I added it using the WebGUI

**Comment by starbasessd** [comment]
Did you try: 
rtsp://192.168.1.76/cam/realmonitor?channel=0&subtype=1
or other channels?

**Comment by arkpoah** [comment]
yes, I get 404 errors in logs on other channels, only channel 1 is working with others rtsp clients.

**Comment by arkpoah** [comment]
I just notice that when motioneye is running, my bandwidth received ~100KB/sec from my cam, so it is possible stream is working well and it is just an issue in visualization in webui ?

**Comment by arkpoah** [comment]
I installed motioneye on my workstation with Manjaro, and it works well, so no problem with camera.
My issue is only when trying to display stream from motioneye installed on Ubuntu 20.04 LTS

Any idea wich package could be in cause?

**Comment by rocket59** [comment]
You need to use this url:

rtsp://<ipaddress>:554/cam/realmonitor?channel=1&subtype=0&unicast=true&proto=Onvif

However the lag on my setup (around 6s) makes the stream pretty much useless. If anyone has this working ok please post, otherwise this camera will be going into storage.
